libdeno/libdeno_test.cc

```diff
@@ -304,7 +304,8 @@ TEST(LibDenoTest, ModuleSnapshot) {
   EXPECT_TRUE(deno_execute_mod(d1, nullptr, "x.js",
                                "const globalEval = eval\n"
                                "const global = globalEval('this')\n"
-                               "global.a = 1 + 2"));
+                               "global.a = 1 + 2",
+                               0));
```

Comment:
Maybe the `is_prefetch` shouldn't be the last argument.. looks kinda wacky hanging off the end there. But I leave it up to your discretion.

Reply:
To me, it sounds like less of an important parameter than other ones on providing info, although it changes what the code functionality. For best readability, we could have made this a separate API...